.azure-pipelines/templates/deploy.yml

```diff
@@ -0,0 +1,120 @@
+parameters:
+- name: release
+  type: boolean
+  default: False
+- name: documentation_artefact_path
+  type: string
+  default: '$(Build.StagingDirectory)/documentation'
+- name: documentation_repo_path
+  type: string
+  default: '$(Build.StagingDirectory)/git'
+- name: image
+  type: string
+  default: ''
+- name: label
+  type: string
+  default: 'test'
```

Comment:
I suppose this should be removed/changed before we merge?

Reply:
i thought it was a good idea to have it as `test`, so that if you mess things up in your setup you don't accidentally push the packages to the main conda label?

Reply:
Not suggestinv that the default should be the main label. Are defaults required? Can we force settings this by not providing a default.

Reply:
there has to be a default, but it can be an empty string, but i think if you provide an empty label to anaconda upload, it might then default to `main` when uploading

Reply:
Ok, fine with leaving it then 👍